reflex/vars.py

```diff
@@ -498,6 +573,22 @@ def __le__(self, other: Var) -> Var:
         """
         return self.compare("<=", other)
 
+    @staticmethod
+    def get_outer_container_type(type_annotation) -> Type:
```

Comment:
I believe you can use the method `types.get_base_class` which should already do this

Reply:
ah yes, good catch